packages/deployments/contracts/contracts/core/connext/facets/AssetFacet.sol

```diff
@@ -166,6 +138,9 @@ contract AssetFacet is BaseConnextFacet {
     address _adoptedAssetId,
     address _stableSwapPool
   ) external onlyOwner {
+    // Native asset support does not exist in this contract
```

Comment:
Can't comment on the Natspec description since it's not part of the PR, but you can remove line 126-128 since native asset is not allowed anymore.